drivers/adc/adc_ite_it8xxx2.c

```diff
@@ -76,17 +83,25 @@ static int adc_it8xxx2_channel_setup(const struct device *dev,
 				     const struct adc_channel_cfg *channel_cfg)
 {
 	const struct adc_it8xxx2_cfg *config = dev->config;
+	uint8_t channel_id = channel_cfg->channel_id;
 
 	if (channel_cfg->acquisition_time != ADC_ACQ_TIME_DEFAULT) {
 		LOG_ERR("Selected ADC acquisition time is not valid");
 		return -EINVAL;
 	}
 
-	if (channel_cfg->channel_id >= CHIP_ADC_COUNT) {
-		LOG_ERR("Channel %d is not valid", channel_cfg->channel_id);
+	/* Support channels 0~7 and 13~16 */
+	if (!((channel_id >= 0 && channel_id <= 7) ||
+	    (channel_id >= 13 && channel_id < 17))) {
```

Comment:
To be consistent with first condition, maybe use channel_id <= 16

Reply:
Done.